features/steps/ruby_notifier_steps.rb

```diff
@@ -12,4 +12,30 @@
   steps %Q{
     When I set environment variable "#{env_var}" to "#{credentials}@#{current_ip}:#{MOCK_API_PORT}"
   }
+end
+Then(/^the "(.+)" of the top project stackframe equals "(.+)"(?: for request (\d+))?$/) do |element, value, request_index|
+  stacktrace = read_key_path(find_request(request_index)[:body], 'events.0.exceptions.0.stacktrace')
+  frame_index = 0
+  stacktrace.each do |frame, index|
+    unless /.*lib\/bugsnag.*\.rb/.match(frame["file"])
```

Comment:
The step definition pattern has "of the top project stackframe", so should this actually check the `inProject` value instead?

Otherwise, if checking against this regular expression is correct, should the step definition pattern have something like "of the top non-Bugsnag stackframe" instead?